org.shiiion.primehack.yml

```diff
@@ -0,0 +1,133 @@
+app-id: org.shiiion.primehack
```

Comment:
there is no `shiiion.org` domain. So this make the app-id invalid as it is not under the developer control.

Reply:
Ah my bad, I misunderstood the app-id. Is a domain necessary for a flatpak? Everything we need is based on github.

Reply:
github is fine. something like `io.github.shiiion.primehack` albeit this is inconsistent with the repository name.